hack/test/cmd/root.go

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "talos-test",
+		Short: "A small cli for talos testing",
```

Comment:
nit: `A CLI for performing tests`. 1 We already know its Talos, 2 "small" isn't that useful :) ... nits I know.